packages/account/src/Components/file-uploader-container/file-uploader.jsx

```diff
@@ -96,11 +96,11 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
                 filename_limit={26}
                 hover_message={localize('Drop files here..')}
                 max_size={max_document_size}
-                message={<UploadMessage />}
+                message={() => <UploadMessage />}
                 multiple={false}
                 onDropAccepted={handleAcceptedFiles}
                 onDropRejected={handleRejectedFiles}
-                validation_error_message={document_file.error_message}
+                validation_error_message={() => document_file.error_message}
```

Comment:
@likhith-deriv i suggest that we don't make these 2 changes in this file and pass < UploadMessage / > to `message` and `document_file.error_message` to `validation_error_message`  the same way as before instead of callbacks 🙂